aws-lambda/jaeger-ruby/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenTracing in AWS Lambda Ruby 2.5
+
+This is an example Lambda function that shows how to use the Ruby Jaeger client to send spans to SignalFx.
+
+The code can be seen in [prime_sum.rb](./prime_sum.rb).
+
+## Using a Deployment Package
+
+A Ruby deployment package has your project's Ruby files and it's gem dependencies.
+
+### Downloading dependencies
+
+The dependencies for this example have already been provided in this repo, so
+the next steps are not necessary unless starting from scratch.
+
+The deployment's `Gemfile` dependencies must be downloaded locally.
+
+```bash
+$ bundle install --path vendor/bundle
+```
+
+This part is only needed if any dependencies have native extensions. The native
```

Comment:
It's not clear to me if that's a required step. You say the dependencies are already provided, but that doesn't tell me if they have any native extensions; so I can't tell whether I need to go through this step when trying out the example.

Reply:
I'll remove this section since we're providing the dependencies. I had that in case the user wanted to download/update the dependencies themselves, but since this is tracing example maybe general Ruby Lambda usage stuff doesn't belong here.

Reply:
I've removed the `vendor` folder instead and just provided a Gemfile. The dependency downloading steps are now required